<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1010</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>3000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>毕业设计 - 基于ARM的上下料机器人控制软件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <widget class="QLabel" name="Lb_Camera_Capture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1280</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid red;</string>
    </property>
    <property name="text">
     <string>未检测到相机</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_dirUp">
    <property name="geometry">
     <rect>
      <x>1710</x>
      <y>270</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>镜头上升
（&gt;）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_dirLeft">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>390</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>镜头左移
（←）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_dirRight">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>390</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>镜头右移
（→）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_dirDown">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>510</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>镜头下降
（&lt;）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_Grab">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>390</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>磨轮
（回车）</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>0</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_13">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QPushButton" name="Bt_Connect">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>连接
机器人</string>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_Push">
         <property name="text">
          <string>发送
指令</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QComboBox" name="Cbox_Camera">
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cbox_Serial"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Bt_CAM_cmd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>880</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>自动对焦</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>120</y>
      <width>86</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="Lb_Target_27">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;传感器状态&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Target_28">
       <property name="text">
        <string>相机 离校/在线</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Target_29">
       <property name="text">
        <string>编码器 NaN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>20</y>
      <width>92</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="Lb_Target_32">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;队列状态&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Target_33">
       <property name="text">
        <string>待处理 NaN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Target_35">
       <property name="text">
        <string>种类 NaN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Target_34">
       <property name="text">
        <string>待拾取 NaN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Target_36">
       <property name="text">
        <string>时间戳 ddhhmmss</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>20</y>
      <width>92</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="Lb_Target_37">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;板卡状态&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Target_38">
       <property name="text">
        <string>温度 NaN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Target_39">
       <property name="text">
        <string>CPU NaN %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Target_40">
       <property name="text">
        <string>RAM NaN %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Target_41">
       <property name="text">
        <string>时间 yyyymmddhh</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Bt_Ctrl_shutdown">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>880</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>控制板关机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_Ctrl_rst">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>880</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>控制板复位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_Ctrl_Cmd">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>880</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>发送指令</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_w">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>430</y>
      <width>130</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>磨轮转速:NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_H">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>400</y>
      <width>130</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>磨轮高度:NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Z">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>440</y>
      <width>130</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>镜头高度:NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Y">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>410</y>
      <width>130</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>镜头Y:NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_X">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>380</y>
      <width>130</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>镜头X:NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Tx">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>875</y>
      <width>680</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Tx Hex</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_Rx">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>801</y>
      <width>680</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Rx Hex</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_dirForward">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>290</y>
      <width>70</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>镜头
前移
（↑）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_dirBackward">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>460</y>
      <width>70</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>镜头
后移
（↓）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_Forward">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>270</y>
      <width>101</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>磨轮
前移
（W）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_Backward">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>510</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>磨轮
后移
（S）</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_20um">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>180</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>20um</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_100um">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>200</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>100um</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_1mm">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>220</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1mm</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_1cm">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>240</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1cm</string>
    </property>
   </widget>
   <widget class="QSlider" name="Sl_StepChose">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>640</y>
      <width>591</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>670</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>20 um</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>670</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>100 um</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>670</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>1 mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1880</x>
      <y>670</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>1 cm</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lb_QoS">
    <property name="geometry">
     <rect>
      <x>1361</x>
      <y>841</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Tx NaN | Rx NaN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mnbr_Dev">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="CameraMenu">
    <property name="title">
     <string>相机</string>
    </property>
    <addaction name="actionDevice_CAM"/>
   </widget>
   <addaction name="CameraMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDevice_CAM">
   <property name="text">
    <string>Device_CAM</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
